s3/templates/cloudtrail.json.tpl

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Is this policy used anywhere? Or is it intended to be referenced by the project that uses the module?

Reply:
It's referenced by the project which uses the module, in this case tdr-aws-accounts, where it's referenced in the cloudtrail_s3 module